README.md

```diff
@@ -0,0 +1,77 @@
+# Maven Plugin Integration Testing
+
+Java library for writing integration tests for maven plugins.
+
+## Features
+
+* Run tests with isolated local repository (the plugin won't be installed in your default local maven repository)
+* Extract code coverage
+* Debugging
+
+## Usage
+
```

Comment:
Add a small note here that it's an example instead of adding it after the code block